week2/homework/src/index.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+let fs = require('fs');
```

Comment:
For variables that do not (or should not) change, prefer using `const` over `let`.
Actually, always try to use `cost`, `let` should be used as little as possible.

This is because if you use `const` you cannot redeclare a variable by mistake and break your app without understanding why. If you reuse a variable you declared with `const` the app will break but you will get an error that tells you why, if you re-assign or re-declare a variable with `let`, the compiler will say nothing but your app might not work as you intended.

Just a pro tip :)

Reply:
Hi @keepitterron 
thanks a lot for your kind recommendations and reviews. 
today I am going to correct the points you mentioned and start to bonus sections. at least I can try:)